iib/workers/tasks/build.py

```diff
@@ -120,7 +121,11 @@ def _cleanup():
 
 
 @retry(exceptions=IIBError, tries=get_worker_config().iib_total_attempts, logger=log)
-def _create_and_push_manifest_list(request_id, arches, build_tags):
+def _create_and_push_manifest_list(
+    request_id: int,
+    arches: Iterable,
```

Comment:
I believe this could be something like this:

```suggestion
    arches: Iterable[str],
```

Reply:
Sure, I missed that somehow.